CONTRIBUTING.md

```diff
@@ -49,17 +49,18 @@ working on your contributions.
 
 ### Format of the Commit Message
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
-the body of the commit should describe the why.
+The commit messages should be designed to answer two questions: what changed and why. 
+
+The subject line should consist of the modified subsytem and an imperative sentence
+describing the what changed and the body of the commit should describe why it changed.
 
 ```
-scripts: add the test-cluster command
+rkt/config: improve configuration tests
 
-this uses tmux to setup a test cluster that you can easily kill and
-start for debugging.
+Tests rkt config pkg.
```

Comment:
I wanted to emphasize `rkt/config` part where you specify the package path if it's a small change, so I changed the body as well, I can expand it.